src/site/includes/breadcrumbs.drupal.liquid

```diff
@@ -1,6 +1,10 @@
 <nav data-template="includes/breadcrumbs" aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
   <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    {% for crumb in entityUrl.breadcrumb %}
+    {% assign crumbs = entityUrl.breadcrumb %}
+    {% if pathBreadcrumb = true %}
```

Comment:
I think you should be using `==` operator for comparisons

Reply:
Thanks for catching that, @ncksllvn !